README.md

```diff
@@ -31,6 +31,34 @@ Then open the generated `./myreport/index.html` file in your browser.
 
 If want, you can [install](https://github.com/phpmetrics/PhpMetrics/blob/master/doc/installation.md) PhpMetrics globally with your favorite package manager. You can also visit our [documentation](http://www.phpmetrics.org/documentation/index.html).
 
+## Configuration file
+
+Use the `--config=<file>.json` option. For example:
+
+```json
+{
+    "includes": [
+        "src"
+    ],
+    "exclude": "tests",
+    "report": {
+        "html": "/tmp/report"
+    },
+    "groups": [
+        {
+            "name": "Component",
+            "match": "!component!i"
+        },
+        {
+            "name": "Example",
+            "match": "!example!"
+        }
+    ]
+}
+```
+
+> In this example, we group "layers" of code ("Component" and "Example" using regex.
```

Comment:
Yes you're right. Using config file is a old (un-documented) feature. I've patched code to use array instead of string for excludes

I've updated the PR with new ewample of configuration